extensions/panache/mongodb-panache-kotlin/deployment/src/test/kotlin/io/quarkus/mongodb/panache/kotlin/deployment/Book.kt

```diff
@@ -0,0 +1,154 @@
+package io.quarkus.mongodb.panache.kotlin.deployment
+
+import com.mongodb.client.MongoCollection
+import com.mongodb.client.MongoDatabase
+import io.quarkus.mongodb.panache.PanacheUpdate
+import io.quarkus.mongodb.panache.kotlin.PanacheMongoCompanion
+import io.quarkus.mongodb.panache.kotlin.PanacheMongoEntity
+import io.quarkus.mongodb.panache.kotlin.PanacheQuery
+import io.quarkus.mongodb.panache.kotlin.runtime.KotlinMongoOperations
+import io.quarkus.mongodb.panache.kotlin.runtime.KotlinMongoOperations.Companion.INSTANCE
+import io.quarkus.panache.common.Parameters
+import io.quarkus.panache.common.Sort
+import org.bson.Document
+import org.bson.types.ObjectId
+import java.util.stream.Stream
+
+@Suppress("UNCHECKED_CAST", "unused")
```

Comment:
You really need to write some explaination of the content of this class so if someone saw the test fail but didn't know the detail of the bytecode enancement knows what's going on.

Reply:
that's a fair point.  i go in to detail in the PR but folks 6 months from now won't know that.  i'll update.